source/common/grpc/grpc_web_filter.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SOURCE_COMMON_GRPC_GRPC_WEB_FILTER_H_
+#define SOURCE_COMMON_GRPC_GRPC_WEB_FILTER_H_
+
+#include "envoy/http/filter.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/grpc/codec.h"
+
+namespace Envoy {
+namespace Grpc {
+
+class GrpcWebFilter : public Http::StreamFilter {
```

Comment:
Can we get some high level doc comments on this filter in terms of what it does, link to the spec, etc. It's very difficult to review the code with no context.